polus-graph-pyramid-builder-plugin/src/requirements.txt

```diff
@@ -2,3 +2,4 @@ pandas==0.25.1
 matplotlib==3.1.1
 numpy==1.17.2
 imageio==2.5.0
+scipy==1.4.1
```

Comment:
I don't believe your code uses scipy. It should be removed since it can add pretty considerably to the size of the Docker container.